com.unity.render-pipelines.universal/Documentation~/writing-shaders-urp-basic-unlit-structure.md

```diff
@@ -0,0 +1,177 @@
+# URP unlit basic shader
+
+This example shows a basic URP-compatible shader. This shader fills the mesh shape with a color predefined in the shader code.
```

Comment:
The first time URP is mentioned on a page, you must write its name as Universal Render Pipeline (URP) - see style guide

Reply:
I've added the full name to the root page (writing-custom-shaders-urp.md) instead.